common/gossmods_listpeerchannels.h

```diff
@@ -30,12 +30,14 @@ struct gossmap_localmods *gossmods_from_listpeerchannels_(const tal_t *ctx,
 								     const struct node_id *self_,
 								     const struct node_id *peer,
 								     const struct short_channel_id_dir *scidd,
-								     struct amount_msat min,
-								     struct amount_msat max,
+								     struct amount_msat htlcmin,
+								     struct amount_msat htlcmax,
+								     struct amount_msat spendable,
 								     struct amount_msat fee_base,
 								     u32 fee_proportional,
 								     u32 cltv_delta,
 								     bool enabled,
+								     bool is_local,
```

Comment:
Hmm, this can be determined by self vs peer: `scidd->dir == node_id_idx(self, peer)`?

Mainly because the `is_local `name is a bit confusing, since it's always a local channel, but you mean "is this information from our local node or remote?".